spring-graal-native-samples/commandlinerunner/compile.sh

```diff
@@ -28,14 +28,15 @@ CP=BOOT-INF/classes:$LIBPATH:$FEATURE
 GRAALVM_VERSION=`native-image --version`
 echo "Compiling $ARTIFACT with $GRAALVM_VERSION"
 time native-image \
+  --verbose \
   --no-server \
   --no-fallback \
   -H:Name=$ARTIFACT \
   -H:+ReportExceptionStackTraces \
   --allow-incomplete-classpath \
   --report-unsupported-elements-at-runtime \
   -DremoveUnusedAutoconfig=true \
-  -cp $CP $MAINCLASS >> output.txt
```

Comment:
By default I think we would like to not "pollute" the output with too many information, could you please elaborate on your need?

Reply:
You are right, I am already outputting the content in case of a failure this is probably not needed